test/Generators/Microsoft.Gen.Logging/Generated/Utils.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Compliance.Redaction;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Telemetry.Logging;
+using Microsoft.Extensions.Telemetry.Testing.Logging;
+using TestClasses;
+
+namespace Microsoft.Gen.Logging.Test;
+
+internal static class Utils
+{
+    private class Provider : ILoggerProvider
+    {
+        private readonly ILogger _logger;
+
+        public Provider(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public ILogger CreateLogger(string categoryName)
+        {
+            return _logger;
+        }
+
+        public void Dispose()
+        {
+            // nothing
+        }
+    }
+
+    public static (ILogger logger, FakeLogCollector collector) GetLogger(out FakeLogger fakeLogger)
```

Comment:
Can we return a wrapper of `ILogger`, something like `IDsisposableLogger`?
I would prefer `serviceCollection.BuildServiceProvider()` to be disposed properly.

Reply:
Done.